src/SolutionTemplate/UnoSolutionTemplate.net6/Shared/App.xaml.cs

```diff
@@ -23,18 +13,13 @@ namespace $ext_safeprojectname$
     /// </summary>
     public sealed partial class App : Application
     {
-#if NET5_0 && WINDOWS
         private Window _window;
```

Comment:
The reason I deleted the conditional directive here is that in `#else` it's using the fully qualified `Windows.UI.Xaml.Window`, which is the same used in `#if` since `using Windows.UI.Xaml` is there.